packages/ui/components/form/date-range-picker/DateRangePicker.tsx

```diff
@@ -63,7 +64,8 @@ export function DatePickerWithRange({
           sideOffset={4}>
           <Calendar
             initialFocus
-            fromDate={minDate}
+            //When explicitly null, we want past dates to be shown as well, otherwise show only dates passed or from current date
```

Comment:
Added comment because this is confusing

Reply:
yeah it seems you are handling null Date and minDate conditions inside the `DateRangePicker`, which is good.